ko/news/_posts/2024-07-09-ruby-3-3-4-released.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: news_post
+title: "Ruby 3.3.4 릴리스"
+author: "k0kubun"
+translator: "shia"
+date: 2024-07-09 00:30:00 +0000
+lang: ko
+---
+
+Ruby 3.3.4가 릴리스되었습니다.
+
+이번 릴리스에는 Ruby 3.3.3의 내장 gem인 `net-pop`, `net-ftp`, `net-imap`, `prime`의
+gemspec에서 의존성이 누락된 회귀 버그를 해결했습니다
+[[Bug #20581]](https://bugs.ruby-lang.org/issues/20581).
+이 수정으로 Bundler가 Heroku와 같은 플랫폼에서 해당 gem을 성공적으로 설치할 수 있습니다.
+만약 `bundle install`이 정상적으로 작동한다면, 이 문제는 해결된 것입니다.
+
+나머지 변경 사항은 대부분 사소한 버그 수정입니다.
+자세한 내용은 [GitHub 릴리스](https://github.com/ruby/ruby/releases/tag/v3_3_4)를 참조하세요.
+
+## 릴리스 일정
+
+앞으로, 최신 안정 버전의 Ruby(현재 Ruby 3.3)를 `.1` 릴리스 이후 2개월마다 릴리스할 계획입니다.
+Ruby 3.3의 경우, 3.3.5는 9월 3일에, 3.3.6은 11월 5일에, 3.3.7은 1월 7일에 릴리스될 예정입니다.
+
+Ruby 3.3.3 사용자 중 Heroku 사용자에 영향을 미치는 릴리스처럼 많은 사람들에게 영향을 미치는 변경 사항이 있을 경우,
```

Comment:
It's quite difficult to translate this in one sentence. How about this?

```suggestion
이번 릴리스는 Heroku의 Ruby 3.3.3 사용자가 영향을 받았습니다. 이와 같이 많은 사람들에게 영향을 미치는 변경 사항이 있을 경우,
```

Reply:
fixed eb00406